plugins/package-managers/nuget/build.gradle.kts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+@Suppress("DSL_SCOPE_VIOLATION") // See https://youtrack.jetbrains.com/issue/KTIJ-19369.
+plugins {
+    // Apply core plugins.
+    `java-library`
+
+    // Apply third-party plugins.
+    alias(libs.plugins.kotlinSerialization)
+}
+
+dependencies {
+    api(project(":analyzer"))
+    api(project(":model"))
+    api(project(":utils:common-utils"))
+
+    api(libs.log4jApiKotlin)
+    api(libs.semver4j)
```

Comment:
@sschuberth we are using the exact same approach than in python-inspector. We are not experts in gradle, so we are not qualified to make changes there. I suggest instead that you make the modifications directly as we will surely botch these. :)

Reply:
> I suggest instead that you make the modifications directly as we will surely botch these. :)

Sure. Just pay me a share of the money you're getting for this 😉

Reply:
When you say:
> This can become an implementation dependency and should be sorted into the according block below.

I do not know what you mean and what to do. Again we do not know gradle.